pkgs/by-name/gr/gradia/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  gtk4,
+  glib,
+  ninja,
+  meson,
+  stdenv,
+  python3,
+  libportal,
+  libadwaita,
+  wrapGAppsHook4,
+  fetchFromGitHub,
+  nix-update-script,
+  blueprint-compiler,
+  desktop-file-utils,
+  gobject-introspection,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gradia";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "AlexanderVanhee";
+    repo = finalAttrs.pname;
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-9iXz4MjgG4VlystZZIKOheK4hYuz6lRXPivy+1K+cYs=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    wrapGAppsHook4
+    blueprint-compiler
+    desktop-file-utils
+    gobject-introspection
+    (python3.withPackages (
+      ps: with ps; [
+        pillow
+        pycairo
+        pygobject3
+        pygobject-stubs
+      ]
+    ))
+  ];
+
+  buildInputs = [
+    glib
+    gtk4
+    libadwaita
+    libportal
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    changelog = "https://github.com/AlexanderVanhee/Gradia/releases/tag/${finalAttrs.version}";
```

Nit/**optional**:
```suggestion
    changelog = "https://github.com/AlexanderVanhee/Gradia/releases/tag/${finalAttrs.src.tag}";
```
Some people prefer one or the other, both is fine. There is benefits to both. You don't have to change anything here, just wanted to point out there might be a different pattern that is emerging here. No docs about it though, so feel free to ignore!

---

I think I'll leave this one as is, ty though!